apps/bfd-model/bfd-model-rda/src/test/java/gov/cms/bfd/model/rda/SchemaMigrationIT.java

```diff
@@ -362,6 +363,29 @@ public void verifyMcsMbiQueries() {
     entityManager.getTransaction().commit();
   }
 
+  @Test
+  public void verifyClaimMetaData() {
```

Comment:
Also missing a doc; please describe what the test does and any nuances about it.

Reply:
Comment has been added.